composer.json

```diff
@@ -14,7 +14,7 @@
     "ext-json": "*",
     "ext-libxml": "*",
     "ext-spl": "*",
-    "ampproject/amp-toolbox": "0.6.0",
+    "ampproject/amp-toolbox": "dev-main",
```

Comment:
Quick question: Would you want to pin it to the commit referred to in bd91615?

Reply:
Yeah, that's probably a good idea.